exporters/sql_export/sql_table_defs.py

```diff
@@ -0,0 +1,168 @@
+from sqlalchemy import Column, ForeignKey, Boolean, Integer, String
```

Comment:
Fin  model bortset fra min generelle ophævelse over it-forbindelser.
Jeg ville foretrække to tabeller:
brugeres_it_systemer
afdelingers_it_systemer